plugins/authenticator/Cargo.toml

```
@@ -23,7 +23,7 @@ once_cell = "1"
 sha2 = "0.10"
 base64 = "0.21"
 u2f = "0.2"
-chrono = "0.4"
+chrono = "=0.4.29"
```

```suggestion
chrono = "<=0.4.29"
```
if we can't support 30 and above we can at least support 28 and below 😂